tests/modules/test_create.py

```diff
@@ -162,3 +163,487 @@ def test_modules_migrate_symlink(self, mock_rich_ask):
+
+        expected_yml = {
+            "name": "bpipe_test",
+            "description": "write your description here",
+            "keywords": ["sort", "example", "genomics"],
+            "tools": [
+                {
+                    "bpipe": {
+                        "description": "",
+                        "homepage": "http://test",
+                        "documentation": "http://test",
+                        "tool_dev_url": "http://test",
+                        "doi": "",
+                        "licence": ["MIT"],
+                        "identifier": "biotools:bpipe",
+                    }
+                }
+            ],
+            "input": [
+                {
+                    "raw_sequence": {
+                        "type": "file",
+                        "description": "raw_sequence file",
+                        "pattern": "*.{fastq-like,sam}",
+                        "ontologies": [
+                            {"edam": "http://edamontology.org/data_0848"},
+                            {"edam": "http://edamontology.org/format_2182"},
+                            {"edam": "http://edamontology.org/format_2573"},
+                        ],
+                    }
+                }
+            ],
+            "output": {
+                "sequence_report": [
+                    {
+                        "*.{html}": {
+                            "type": "file",
+                            "description": "sequence_report file",
+                            "pattern": "*.{html}",
+                            "ontologies": [
+                                {"edam": "http://edamontology.org/data_2955"},
+                                {"edam": "http://edamontology.org/format_2331"},
+                            ],
+                        }
+                    }
+                ],
+                "versions": [
+                    {
+                        "versions.yml": {
+                            "type": "file",
+                            "description": "File containing software versions",
+                            "pattern": "versions.yml",
+                            "ontologies": [{"edam": "http://edamontology.org/format_3750"}],
+                        }
+                    }
+                ],
+            },
+            "authors": ["@author"],
+            "maintainers": ["@author"],
+        }
+
+        with open(Path(self.nfcore_modules, "modules", "nf-core", "bpipe", "test", "meta.yml")) as fh:
+            meta_yml = yaml.safe_load(fh)
+
+        assert meta_yml == expected_yml
+
+    @mock.patch("nf_core.utils.anaconda_package")
+    @mock.patch("nf_core.utils.get_biocontainer_tag")
+    @mock.patch("nf_core.components.components_utils.get_biotools_response")
+    @mock.patch("rich.prompt.Confirm.ask")
+    def test_modules_meta_yml_structure_template_meta(
+        self, mock_rich_ask, mock_biotools_response, mock_biocontainer_tag, mock_anaconda_packate
```

Comment:
```suggestion
        self, mock_rich_ask, mock_biotools_response, mock_biocontainer_tag, mock_anaconda_package
```
same typo afterwards 🙂